cmd/crc/cmd/status.go

```diff
@@ -114,7 +128,7 @@ func createBars(cpuUse []int64, writer io.Writer) (ramBar *pb.ProgressBar, cpuBa
 	ramBar.SetWriter(writer)
 	ramBar.Set(pb.Bytes, true)
 	ramBar.Set(pb.Static, true)
-	tmpl := `{{ red "RAM:" }} {{counters . }} {{percent .}} {{string . "my_green_string" | green}} {{ bar . "[" "\u2588" "\u2588" " " "]"}} `
+	tmpl := `{{ red "RAM:" }} {{counters . }} {{percent .}} {{ bar . "[" "\u2588" "\u2588" " " "]"}} `
```

Comment:
This removal of `{{string . "my_green_string" | green}}` is because of the issue or we don't need it anymore?

Reply:
Removed it as it is a typo and not required for printing.